src/defs/clientPoliciesRepresentation.ts

```diff
@@ -0,0 +1,9 @@
+import ClientPolicyRepresentation from './clientPolicyRepresentation';
+
+/**
+ * https://www.keycloak.org/docs-api/15.0/rest-api/#_clientpoliciesrepresentation
+ */
+
```

Comment:
Can you remove this newline so the comment is aligned with the interface? That way it will be picked up as JSDoc.